gap/SimplicialCategory.gi

```diff
@@ -21,23 +21,23 @@ InstallGlobalFunction( SimplicialCategoryTruncatedInDegree,
     elif n = 1 then
         F := FreeCategory( RightQuiver( "Delta(C0,C1)[id:C1->C0,s:C0->C1,t:C0->C1]" ) );
         return F /
-               [ [ F.s * F.id, IdentityMorphism( F.C0 ) ],
```

Comment:
Why were these changes necessary and part of this PR?

Reply:
`\*` for two morphisms is installed in CategoryConstructor, but CategoryConstructor is not a hard dependency (only a transitively suggested one). Before this PR, FreydCategoriesForCAP was loaded including suggested dependencies, which pulled in CategoryConstructor. With this PR, suggested dependencies are not loaded anymore.